### ISSUE TITLE
Train Samples problem?

### ISSUE THREAD

**Issue by feixuedu1106** [AlexeyAB/darknet#806]
I want use YOLO to train myself samples. but when I train 800 images, It is very well. but when I add 5000 images, train error like the show. I don't know why the error like this. would you give me some advice.
![all curves3](https://user-images.githubusercontent.com/6083782/39973795-5290bb60-5756-11e8-8cf1-53897fc7048b.png)

**Comment by AlexeyAB** [comment]
@feixuedu1106 
* Did you use Yolo_mark to label your dataset?
* What did you change in the cfg-file?
* What params did you use in the Makefile?
* Do you use 1 GPU or multi-gpu?

**Comment by feixuedu1106** [comment]
1. Yes, I used Yolo_mark label the image. I have two different samples, the first only have 800 images, and train it very well. I did't changle cfg file ,when I train other sample have 5000 images , the train result like upper photo.
2. did't change cfg file. same as when I train 800 images.
3.  
4. just one GPU. GTX1060.

At the moment I suspect that it is a sample problem， So I divided the sample into multiple copies and trained them separately to assess which picture have problem.

**Comment by AlexeyAB** [comment]
It seems some images have wrong annotations.
Can you get good mAP > 50%?

**Comment by feixuedu1106** [comment]
Yes, train 800 images model Precision 80%,recall:78% in some threshold. so the mAP  > 50%. I did't test it.

**Comment by feixuedu1106** [comment]
![all curves2](https://user-images.githubusercontent.com/6083782/40090700-112a762a-58e6-11e8-9bb6-d0ccb10b141b.png)
I just train 2500 samples, and It show again.

**Comment by feixuedu1106** [comment]
![all curves3](https://user-images.githubusercontent.com/6083782/40090767-70819d4c-58e6-11e8-8990-168ad9bf7793.png)
I just train 2500 samples, and it show again. I do't know why? label sample error? code error? or some others.

**Comment by AlexeyAB** [comment]
Try to get mAP using this repo: https://github.com/AlexeyAB/darknet#when-should-i-stop-training
`./darknet detector map data/obj.data yolo-obj.cfg backup/yolo-obj_2500.weights`
